packages/fxa-auth-server/lib/routes/recovery-codes.js

```diff
@@ -309,6 +309,7 @@ module.exports = (log, db, config, customs, mailer, glean, statsd) => {
           id: tokenId,
           tokenVerificationId,
           uid,
+          sessionToken,
```

Comment:
This does not seem correct... The credentials object is a SessionToken class instance...

Reply:
Thanks, for some reason this seemed to work when I tested it locally but I'm running into an issue with a test and I think that's probably it.